specificatie/persoon.yaml

```diff
@@ -10,12 +10,15 @@ components:
       description: |
         Het A-nummer van de persoon
       type: string
+      pattern: ^[0-9]{10}$
     Burgerservicenummer:
       type: string
       pattern: ^[0-9]{9}$
       example: "555555021"
     GbaGeheimhoudingPersoonsgegevens:
       type: integer
+      minimum: 0
```

Comment:
hier vraag ik me af of minimum en maximum nodig is. opgeven dat het een integer is, is al behoorlijk strikt.